### DISCUSSION TITLE
I can't access admin console

### DISCUSSION THREAD

**Discussion by billypon** [keycloak/keycloak#35534] [Q&A]
now I can't access my admin console.
webui shows me an error `Invalid username or password` directly, but i don't enter username and password yet, and i have cleaned all caches/storages/cookies.

clients are fine, except admin console(client id: security-admin-console).
what can i do fo solving this problem?

i got an console error:

> keycloak-1  | 2024-12-03 05:21:13,511 WARN  [org.keycloak.services] (executor-thread-2) KC-SERVICES0013: Failed authentication: org.keycloak.authentication.AuthenticationFlowException: Invalid client or Invalid client credentials
> keycloak-1  |   at org.keycloak.authentication.ClientAuthenticationFlow.processFlow(ClientAuthenticationFlow.java:111)
> keycloak-1  |   at org.keycloak.authentication.AuthenticationProcessor.authenticateOnly(AuthenticationProcessor.java:1074)
> keycloak-1  |   at org.keycloak.authentication.AuthenticationProcessor.authenticate(AuthenticationProcessor.java:912)
> keycloak-1  |   at org.keycloak.protocol.AuthorizationEndpointBase.handleBrowserAuthenticationRequest(AuthorizationEndpointBase.java:147)
> keycloak-1  |   at org.keycloak.protocol.oidc.endpoints.AuthorizationEndpoint.buildAuthorizationCodeAuthorizationResponse(AuthorizationEndpoint.java:353)
> keycloak-1  |   at org.keycloak.protocol.oidc.endpoints.AuthorizationEndpoint.process(AuthorizationEndpoint.java:204)
> keycloak-1  |   at org.keycloak.protocol.oidc.endpoints.AuthorizationEndpoint.buildGet(AuthorizationEndpoint.java:115)
> keycloak-1  |   at org.keycloak.protocol.oidc.endpoints.AuthorizationEndpoint$quarkusrestinvoker$buildGet_4b690b27439f19dd29733dc5fd4004f24de0adb6.invoke(Unknown Source)
> keycloak-1  |   at org.jboss.resteasy.reactive.server.handlers.InvocationHandler.handle(InvocationHandler.java:29)
> keycloak-1  |   at io.quarkus.resteasy.reactive.server.runtime.QuarkusResteasyReactiveRequestContext.invokeHandler(QuarkusResteasyReactiveRequestContext.java:141)
> keycloak-1  |   at org.jboss.resteasy.reactive.common.core.AbstractResteasyReactiveContext.run(AbstractResteasyReactiveContext.java:147)
> keycloak-1  |   at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:635)
> keycloak-1  |   at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
> keycloak-1  |   at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
> keycloak-1  |   at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1495)
> keycloak-1  |   at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
> keycloak-1  |   at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
> keycloak-1  |   at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
> keycloak-1  |   at java.base/java.lang.Thread.run(Thread.java:1583)
> keycloak-1  | 
> keycloak-1  | 2024-12-03 05:21:13,549 WARN  [org.keycloak.events] (executor-thread-2) type="LOGIN_ERROR", realmId="791dddc0-5f88-4e21-9439-4b8ab5b51c62", realmName="master", clientId="security-admin-console", userId="null", ipAddress="172.19.0.8", error="invalid_user_credentials", auth_method="openid-connect", auth_type="code", response_type="code", redirect_uri="/admin/master/console/", code_id="5b23d07b-4e20-4233-b97a-8e8c1ad79587", response_mode="query"

my service is running as docker container, here's startup message:
 
> keycloak-1  | Updating the configuration and installing your custom providers, if any. Please wait.
> keycloak-1  | 2024-12-03 05:17:25,343 INFO  [io.qua.dep.QuarkusAugmentor] (main) Quarkus augmentation completed in 12239ms
> keycloak-1  | 2024-12-03 05:17:27,519 INFO  [org.keycloak.url.HostnameV2ProviderFactory] (main) If hostanme is specified, hostname-strict is effectively ignored
> keycloak-1  | 2024-12-03 05:17:31,048 INFO  [org.keycloak.quarkus.runtime.storage.infinispan.CacheManagerFactory] (main) Starting Infinispan embedded cache manager
> keycloak-1  | 2024-12-03 05:17:31,341 INFO  [org.keycloak.quarkus.runtime.storage.infinispan.CacheManagerFactory] (main) Persistent user sessions enabled and no memory limit found in configuration. Setting max entries for sessions to 10000 entries.
> keycloak-1  | 2024-12-03 05:17:31,342 INFO  [org.keycloak.quarkus.runtime.storage.infinispan.CacheManagerFactory] (main) Persistent user sessions enabled and no memory limit found in configuration. Setting max entries for clientSessions to 10000 entries.
> keycloak-1  | 2024-12-03 05:17:31,342 INFO  [org.keycloak.quarkus.runtime.storage.infinispan.CacheManagerFactory] (main) Persistent user sessions enabled and no memory limit found in configuration. Setting max entries for offlineSessions to 10000 entries.
> keycloak-1  | 2024-12-03 05:17:31,342 INFO  [org.keycloak.quarkus.runtime.storage.infinispan.CacheManagerFactory] (main) Persistent user sessions enabled and no memory limit found in configuration. Setting max entries for offlineClientSessions to 10000 entries.
> keycloak-1  | 2024-12-03 05:17:32,141 INFO  [org.infinispan.CONTAINER] (ForkJoinPool.commonPool-worker-1) ISPN000556: Starting user marshaller 'org.infinispan.commons.marshall.ImmutableProtoStreamMarshaller'
> keycloak-1  | 2024-12-03 05:17:33,891 INFO  [org.keycloak.broker.provider.AbstractIdentityProviderMapper] (main) Registering class org.keycloak.broker.provider.mappersync.ConfigSyncEventListener
> keycloak-1  | 2024-12-03 05:17:33,946 INFO  [org.keycloak.connections.infinispan.DefaultInfinispanConnectionProviderFactory] (main) Node name: node_339168, Site name: null
> keycloak-1  | 2024-12-03 05:17:35,382 WARN  [io.agroal.pool] (main) Datasource '<default>': JDBC resources leaked: 1 ResultSet(s) and 0 Statement(s)
> keycloak-1  | 2024-12-03 05:17:35,689 INFO  [io.quarkus] (main) Keycloak 26.0.5 on JVM (powered by Quarkus 3.15.1) started in 10.125s. Listening on: http://0.0.0.0:8080
> keycloak-1  | 2024-12-03 05:17:35,690 INFO  [io.quarkus] (main) Profile dev activated. 
> keycloak-1  | 2024-12-03 05:17:35,690 INFO  [io.quarkus] (main) Installed features: [agroal, cdi, hibernate-orm, jdbc-mysql, keycloak, narayana-jta, opentelemetry, reactive-routes, rest, rest-jackson, smallrye-context-propagation, vertx]
> keycloak-1  | 2024-12-03 05:17:35,697 WARN  [org.keycloak.quarkus.runtime.KeycloakMain] (main) Running the server in development mode. DO NOT use this configuration in production.

**Comment by billypon** [comment]
i have handled it!

**Reply by lovrodoe** [reply]
How? I have the same issue

**Reply by billypon** [reply]
I don't known what has happended.
I connected to mysql server, backed up all data, and re-deployed keycloak instance.
Then I restored all users' and clients' data manually.
